MCMC/utils.py

```diff
@@ -2,6 +2,33 @@
 import numpy as np
 from numba import jit
 
+def avg_nn(sim_obj, n_frames, r_max=3):
+    box = [sim_obj.L, sim_obj.L,0]
+    frame_avg_nn = []
+    for points in sim_obj.system_history[-n_frames:]:
+        points = np.append(
+            points, np.zeros((points.shape[0], 1)),axis=1
```

Comment:
What is the np.zeros((points.shape[0],1)) doing?

Reply:
Our system is 2D, which means all of our coordinates are only (x, y). Sometimes we need 3 dimensional coordinates to use other tools, like freud, or writing to a gsd file. This line adds a z coordinate of 0 to all of our 2D coordinates.